mercadoLibre/mercadoLibre/DetailViewController.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  DetailViewController.swift
+//  mercadoLibre
+//
+//  Created by Jurymar Colmenares on 4/12/24.
+//
+import UIKit
+
+class DetailViewController: UIViewController {
+    
+    // 1. Propiedad que contiene el elemento seleccionado de la lista.
+    var item: Item
+    
+    // 2. Elementos visuales: etiquetas para el título, la imagen y el precio.
+    let titleLabel = UILabel()
+    let imageView = UIImageView()
+    let priceLabel = UILabel()
```

Comment:
Better to have UI children as private
```suggestion
   private let titleLabel = UILabel()
   private let imageView = UIImageView()
   private let priceLabel = UILabel()
```